endpointmanager/pkg/nppesquerier/populatenpiorgs.go

```diff
@@ -433,7 +447,7 @@ func ParseAndStoreNPIFile(ctx context.Context, fname string, store *postgresql.S
 		}
 
 		if i%10000 == 0 {
-			log.Infof("Processed %d/%d NPI entities. Added %d organizations.\n", i, len(lines), added)
+			log.Infof("Processed %d/%d NPI entities. Added %d organizations.\n", i, len(line), added)
```

Comment:
Just changed this so it prints "Process %d NPI entities" instead, since the only way to know the total number of lines is opening the entire file at once